src/linear_algebra/quadratic_form.lean

```diff
@@ -184,12 +188,13 @@ instance : add_comm_monoid (quadratic_form R M) :=
   ..quadratic_form.has_add,
   ..quadratic_form.has_zero }
 
-instance : distrib_mul_action R₁ (quadratic_form R₁ M) :=
-{ mul_smul := λ a b Q, ext (λ x, by simp [mul_smul, smul_comm]),
+instance : semimodule R₁ (quadratic_form R₁ M) :=
```

Comment:
It's actually a module, right? You have a negation on `quadratic_form _ _`.

Reply:
You are completely right! I was making the same mistake as for `has_scalar`.

Reply:
Awesome! Thanks!